### PR TITLE
Add 2.0-preview branch information to the README

### PR DESCRIPTION
**Pull request by erinha**
Add 2.0-preview branch information to the README that is published with the npm package. Also cleaned up Getting Started commands.

### CODE REVIEW

**Comment by erinha** [comment]
<p>Analyzed commit id : 8b0141cfca1a511d411c5ed466b68f77b63829b4</p><hr><details><summary><b><span style="color: green">⯆</span> @microsoft/bundle-analysis-app:</b> -7 Bytes</summary><table><tr><th>Metric Name</th><th>Baseline Size</th><th>Compare Size</th><th>Size Diff</th></tr><tr><td>main.js</td><td>58.4 KB</td><td>58.4 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr><tr><td>Total Size</td><td>58.4 KB</td><td>58.4 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr></table></details><hr><p>Baseline commit: c6c06472870b928fb55e92be2d1e3d7630a282ee</p>

File: README.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ This JavaScript library is part of the [Microsoft Teams developer platform](http
 ## Getting Started
 
 1. Clone the repo
-2. `yarn install` from repo root
-2. `yarn build` from repo root
-3. to run Unit test `yarn test`
+2. Run `yarn install` from repo root
+2. Run `yarn build` from repo root
```
Comment (MarcinJuraszek) [inline README.md:14]: while you're here, can you fix the numbering?

Comment (erinha) [reply README.md]: Done.

**Review by TrevorJoelHarris — APPROVED** [review]
:shipit:

**Comment by erinha** [comment]
<p>Analyzed commit id : 225c660d08602bbee8a8e611c8fcdc4a6353192a</p><hr><details><summary><b><span style="color: green">⯆</span> @microsoft/bundle-analysis-app:</b> -7 Bytes</summary><table><tr><th>Metric Name</th><th>Baseline Size</th><th>Compare Size</th><th>Size Diff</th></tr><tr><td>main.js</td><td>58.94 KB</td><td>58.93 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr><tr><td>Total Size</td><td>58.94 KB</td><td>58.93 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr></table></details><hr><p>Baseline commit: f33919b6b97b4d1836bce2dc8f5f50d8b0190c29</p>

**Review by MarcinJuraszek — APPROVED** [review]

**Comment by erinha** [comment]
<p>Analyzed commit id : ac855e33df607d1d96338b1e2ffcecfe661d9878</p><hr><details><summary><b><span style="color: green">⯆</span> @microsoft/bundle-analysis-app:</b> -7 Bytes</summary><table><tr><th>Metric Name</th><th>Baseline Size</th><th>Compare Size</th><th>Size Diff</th></tr><tr><td>main.js</td><td>59.19 KB</td><td>59.19 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr><tr><td>Total Size</td><td>59.19 KB</td><td>59.19 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr></table></details><hr><p>Baseline commit: fe4bd57473d755499fb0320b67de8a0beeda8e72</p>

**Pull request merged**